### PR TITLE
Add Status to Storybook

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Relevant documentation updated
- [x] Prettier run on changed files

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
     "@storybook/addon-links": "^5.3.17",
     "@storybook/addons": "^5.3.17",
     "@storybook/react": "^5.3.17"
+  },
+  "dependencies": {
+    "@backstage/theme": "^0.1.1-alpha.3"
```
Comment (Rugvip) [inline packages/storybook/package.json:22]: is this actually needed? O_o

Comment (stefanalund) [reply packages/storybook/package.json:22]: @Rugvip without it the running Storybook locally does not work. Feel free to verify on your end!

Comment (stefanalund) [reply packages/storybook/package.json:22]: Does it work for you?

Comment (stefanalund) [reply packages/storybook/package.json:22]: @Rugvip this was fixed in #539 so I’ll remove this bit.

**Pull request closed without merging**